.vscode/settings.json

```diff
@@ -1,5 +1,6 @@
 {
     "editor.detectIndentation": false,
     "editor.insertSpaces": true,
-    "editor.tabSize": 4
+    "editor.tabSize": 4,
+    "typescript.tsdk": "node_modules\\typescript\\lib"
```

Comment:
This helps VSCode to not have inconsistencies between installed TS version in VSCode and installed TS version in the project.